packages/components/src/components/hds/reveal/index.hbs

```diff
@@ -7,7 +7,7 @@
   <:toggle as |t|>
     <Hds::Reveal::Toggle::Button
       aria-controls={{this.contentId}}
-      @text={{if (and t.isOpen @textWhenOpen) @textWhenOpen this.text}}
+      @text={{this.getText t.isOpen @text @textWhenOpen}}
```

Comment:
thought: could we simplify `this.getText` to get only the required argument `t.isOpen`? `@text` and `@textWhenOpen` can be retrieved via the `args` object in the backing class, right?

Reply:
yeah, good point! 👍

Reply:
Done